content/articles/2018-07-broom-0-5-0.Rmarkdown

```diff
@@ -0,0 +1,127 @@
+---
+title: broom 0.5.0
+author: Alex Hayes
+date: "2018-07-18"
+slug: broom-0-5-0
+description: > 
+  broom 0.5.0 is on CRAN.
+categories:
+  - package
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+
+options(tibble.print_min = 5, tibble.print_max = 5)
+```
+
+We are delighted to announced that [broom](https://broom.tidyverse.org/) 0.5.0 is now available on CRAN. broom 0.5.0 is a major new release featuring new changes that affect both users and developers. See the [News]( https://broom.tidyverse.org/news/) for a detailed list of changes.
+
+## Tibble output
+
+All tidiers should now return `tibble`s rather than `data.frame`s. This allows broom to take advantage of the nice tibble print method and the more consistent behavior of tibbles:
+
+```{r}
+library(broom)
+
+fit <- lm(mpg ~ ., mtcars)
+tidy(fit)
+```
+
+
+These changes will mostly likely affect you when:
+
+- subset with `[`, which always returns a tibble
+- set rownames on a tibble, which is deprecated
+- use augment methods on models with matrix covariates specified in a formula, which will error
+
+Augment will error with matrix covariates because tibbles are more strict about their contents than data frames. Data frames allow users to specify columns in a matrix, like so:
```

Comment:
Augment methods — that way it's a proper subject. Or `augment()`, if you're referring to the function.